misk-redis/src/testFixtures/kotlin/misk/redis/testing/FakeRedis.kt

```diff
@@ -539,6 +568,146 @@ class FakeRedis @Inject constructor(
 
   override fun zscore(key: String, member: String): Double? {
     if (sortedSetKeyValueStore[key] == null) return null
-    return sortedSetKeyValueStore[key]!!.data[member]
+
+    var currentScore:Double? = null
+    for (entries in sortedSetKeyValueStore[key]!!.data.entries) {
+      if (entries.value.contains(member)) {
+        currentScore = entries.key
+        break
+      }
+    }
+
+    return currentScore
+  }
+
+  override fun zrange(
+    key: String,
+    type: ZRangeType,
+    start: ZRangeMarker,
+    stop: ZRangeMarker,
+    reverse: Boolean,
+    limit: ZRangeLimit?,
+  ): List<ByteString?> {
+    return zrangeWithScores(key, type, start, stop, reverse, limit)
+      .map { (member, _) ->  member }.toList()
```

Comment:
My guess is that we're not re-using `ZRangeParams` because it's not possible to inspect the parameters and provide a fake `zrange` implementation. I actually think this is preferred, i.e. throw NotImplemented in all the zrange functionality. The implementation is too complex for a fake. Folks should not be testing this stuff with a fake, they should be testing again Docker.

Reply:
Spinning up Docker slows down tests. So I think it's reasonable for folks to use a fake for trivial commands. But the z-family of commands... they should probably all be tested again Docker

Reply:
To be fair, there are some other `FakeRedis` methods that should throw NotImplemented but don't, e.g. [watch and unwatch](https://github.com/cashapp/misk/blob/29e65e5836bd6c973a98e2343dbd4a060f5cdab2/misk-redis/src/testFixtures/kotlin/misk/redis/testing/FakeRedis.kt#L402-L408). We need to have a bigger discussion about fake vs. Docker, and that's beyond the scope of this PR.